app/src/components/learningJourney/details/LJDeletionPopUp.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from "react";
+import { deleteLearningJourneyWithLJId } from "src/api/learningJourney";
+
+function LJDeletionPopUp({ljId}) {
+    // Hide the popup when click on cancel
+    const [isOpen, setIsOpen] = useState(true);
+    const onClick = () => {setIsOpen(!isOpen)}
+
+    if(!isOpen){
+        return null
+    }
+
+    const DeleteLJButtonClick = (e) => {
+        e.stopPropagation();
+        const res = deleteLearningJourneyWithLJId(ljId);
+        console.log(res);
```

Comment:
/nit remove if no longer necessary

Reply:
Seems necessary. Remember that functions should be camel case so deleteLJ...